test/test.spec.ts

```diff
@@ -20,6 +20,34 @@ declare global {
   }
 }
 
+function ValidateDashboardConfigurationWorksAsExpected(pageView: string, exceptionExpected: boolean, powerbi: service.Service) {
+  const embedUrl = `https://app.powerbi.com/dashboardEmbed`;
+  const component = $(`<div powerbi-embed-url="${embedUrl}" powerbi-type="report"></div>`)
+    .appendTo('#powerbi-fixture');
+
+  const dashboardEmbedConfig = {
+    type: "dashboard",
+    id: "fakeReportId",
+    accessToken: "fakeAccessToken",
+    embedUrl: "fakeEmbedUrl",
+    pageView: pageView
+  };
+
+  powerbi.embed(component[0], <any>dashboardEmbedConfig);
```

Comment:
The validate is called only on "load" (not on "embed"). In "reality" - the constructor of embed class assigns "onLoad" to the iframe that calls the "load" function when the iframe is ready. That doesn't happen on tests, so we need to call the embed again - when embeding the second time the code flow doesn't wait for the iframe to be ready and calls the load function.